.changeset/chilled-bags-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/gen-wrapper-react': patch
+---
+
+Added golden tests.
```

Comment:
This is what will show up in the change log... should it just be empty?

Reply:
Done